redpanda-apicurio/examples/python/kafka-produce-avro.py

```diff
@@ -41,7 +41,7 @@
 # The last command above should create a table with columns: [ KafkaPartition, KafkaOffset, KafkaTimestamp, Symbol, Price ]
 #
 # Run this script on the host (not on a docker image) to generate one row:
-# $ python3 ./kafka-produce-avro.py share_price avro/share_price.json str:Symbol=MSFT str:Side=BUY double:Price=274.82 int:Qty=200
+# $ python3 ./kafka-produce-avro.py share_price 0 avro/share_price.json str:Symbol=MSFT str:Side=BUY double:Price=274.82 int:Qty=200
```

Comment:
I wonder if we should start using argparse here to avoid making these examples too cumbersome.

Reply:
I don't want to re-write the examples in this PR. Happy if @jcferretti  wants to do so in a subsequent one.

Reply:
Not accepting "thrown under the bus offer" on this one.  Either this change is at a good enough usability threshold or not.  Whoever breaks the usability good enough threshold fixes it.  In the same PR or a subsequent one as a result of a negotiation with their reviewers.  I propose Charles makes a call on whether this is breaking the usability threshold or not.